data/json/furniture.json

```diff
@@ -835,7 +835,7 @@
     "id": "f_exercise",
     "name": "exercise machine",
     "symbol": "T",
-    "description": "Typically used for, well, exercising.  You're not up for it.",
+    "description": "Typically used for, well, exercising.  You're getting quite enough of that, running for your life.",
```

Comment:
```suggestion
    "description": "Typically used for, well, exercising.  You're getting quite enough of that; running for your life.",
```

Reply:
> 

Technically, "running for your life" is a partial phrase (I'm blanking on what it's actually called), that's dependent on the previous part of the sentence. It doesn't stand on its own.

Reply:
Both are complete phrases, and checking [this source](https://www.grammarly.com/blog/semicolon/) seems like an accurate use.

What you are doing is called a "[comma splice](https://en.wikipedia.org/wiki/Comma_splice)", which can be both an error or informal writing. However, in consideration for translators (which may not be an issue but I tend to err on caution as I *don't* speak [non-SVO languages](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object)), it's typically better to avoid too much slang/informal writing as it can be difficult to convey in other languages accurately.